src/iroh.udl

```diff
@@ -15,7 +15,7 @@ enum LogLevel {
 
 interface IrohNode {
   [Throws=IrohError]
-  constructor();
+  constructor(string path);
```

Comment:
what do we do when we're in an FFI context without access to a filesystem? empty string as an "in memory" convention?

Generally it strikes me that the constructor args are going to blow out a bunch. When is the right time to go to a configuration struct?

Reply:
> When is the right time to go to a configuration struct?

later 😅  

> what do we do when we're in an FFI context without access to a filesystem?

that's just not supported right now, even the existing version right now is only disk based

Reply:
dope. These are all the answers I needed